Uno.Material.sln

```diff
@@ -30,13 +30,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{
 	EndProjectSection
 EndProject
 Global
-	GlobalSection(SharedMSBuildProjectFiles) = preSolution
```

Comment:
that's a weird change ?

Reply:
Oh yeah I don't know where this is coming from.